column-grouped/columnGroupedChart.js

```diff
@@ -115,6 +132,10 @@ export function draw() {
         }
         return chart;
     };
+    chart.numbers = (d) => {
```

Comment:
In general we should generally make setters also getters, it's an extra line and improves interoperability down the line.

Reply:
You mean